db/fixtures/ae_datastore/ManageIQ/Cloud/VM/Provisioning/StateMachines/ProvisionRequestApproval.class/__methods__/validate_request.rb

```diff
@@ -6,6 +6,35 @@
 # category values: [prov_max_cpu, prov_max_vm, prov_max_memory, prov_max_retirement_days]
 #
 
+def vmdb_object(model, id)
+  $evm.vmdb(model, id.to_i) if model && id
+end
+
+def requested_memory(prov_resource, number_of_vms)
+  memory = get_option_value(prov_resource, :memory) || flavor_value(prov_resource, :memory)
```

Comment:
If `get_option_value(prov_resource, :memory)` is still required it should use `:vm_memory` which is the proper field name.